module1.js

```diff
@@ -80,7 +90,10 @@ function question5(number1, number2) {
 */
 
 function question6(array) {
-  // TODO:
+  numberOfElements = array.length;
+  for(var i = 0; array.length > i; i++){
+    return array[i];
```

Comment:
Again you're returning only one element in the array - and the output of this function is meant to be a string. If you need to list all elements in an array within one string, you can't really do this with a loop. There's one handy function you can use on arrays to list all elements though :)